aries-site/src/components/cards/CardGrid.js

```diff
@@ -12,11 +14,16 @@ export const CardGrid = ({ children, ...rest }) => {
       fill
       {...rest}
     >
-      {children}
+      {cards &&
+        cards.map(topic => <LinkedCard key={topic.name} topic={topic} />)}
```

Comment:
Can we just use ContentCard here and wrap the Link around it? Feels like `LinkedCard` is an over abstraction.

Reply:
Good call. Will re-evaluate.

Reply:
Implemented.